File: samples/client/petstore/java/src/main/java/org/openapijsonschematools/components/schemas/BooleanSchema.java
package org.openapijsonschematools.components.schemas;
import org.openapijsonschematools.configurations.SchemaConfiguration;
import org.openapijsonschematools.schemas.JsonSchema;
import org.openapijsonschematools.schemas.JsonSchemas;
import org.openapijsonschematools.schemas.FrozenList;
import org.openapijsonschematools.schemas.FrozenMap;

import java.time.LocalDate;
import java.time.ZonedDateTime;
import java.util.AbstractMap;
import java.util.LinkedHashMap;
import java.util.LinkedHashSet;
import java.util.List;
import java.util.Map;
import java.util.Set;

public class BooleanSchema {
    // nest classes so all schemas and input/output classes can be public
    
    
    public class BooleanSchema1 implements JsonSchema {
        public static Boolean validate(Boolean arg, SchemaConfiguration configuration) {
            return JsonSchema.validate(JsonSchemas.BooleanSchema.class, arg, configuration);
        }
    }
}

File: samples/client/petstore/kotlin/src/main/kotlin/org/openapijsonschematools/client/components/schemas/BooleanSchema.kt
package org.openapijsonschematools.client.components.schemas;
import org.checkerframework.checker.nullness.qual.Nullable;
import org.openapijsonschematools.client.schemas.BooleanJsonSchema;

public class BooleanSchema extends BooleanJsonSchema {
    // nest classes so all schemas and input/output classes can be public
    
    
    public static class BooleanSchema1 extends BooleanJsonSchema.BooleanJsonSchema1 {
        private static @Nullable BooleanSchema1 instance = null;
        public static BooleanSchema1 getInstance() {
            if (instance == null) {
                instance = new BooleanSchema1();
            }
            return instance;
        }
    }

}
